internal/auth/mux.go

```diff
@@ -60,6 +60,25 @@ func NewAuthenticatorMux(opts *Options, statsdClient *statsd.Client) (*Authentic
 			fmt.Sprintf("/%s/", idpSlug),
 			http.StripPrefix(fmt.Sprintf("/%s", idpSlug), authenticator.ServeMux),
 		)
+
```

Comment:
can we add something to indicate that this is temporary and that we want to take this out eventually?

Reply:
Added!